forms/static/forms/admin/dependent_fields.js

```diff
@@ -50,15 +50,24 @@
         $('.field-item_number').hide();
         $('.field-webpage_layer_no').hide();
         $('.field-age').hide();
+        // Make the Does the story identify the person as either a victim or survivor? question false by default
+        $('#id_newspaperperson_set-0-victim_or_survivor_1').prop("checked", true);
+        $('#id_radioperson_set-0-victim_or_survivor_1').prop("checked", true);
+        $('#id_televisionperson_set-0-victim_or_survivor_1').prop("checked", true);
+        $('#id_internetnewsperson_set-0-victim_or_survivor_1').prop("checked", true);
+        // Select 'Do not Know for the  Is there a photograph of the person in the story?' question since it's required.
+        $("#id_newspaperperson_set-0-is_photograph option[value=3]").prop("selected", true);
+        $("#id_internetnewsperson_set-0-is_photograph option[value=3]").prop("selected", true);
+        $("#id_twitterperson_set-0-is_photograph option[value=3]").prop("selected", true);
```

Comment:
Why do we need to pre-select them? I'd be happy if the user has to do the selection themselves... every other field say in **Basic information** tab is required but we don't pre-populated/select it.

Reply:
That's true. But for something like **Basic information**, the system will show the user errors and the user can fix them. For a tab like **People in the news** where we have to hide some fields from the user, the user will not have the option to select these hidden fields, and since they are required in the backend, the form won't submit but will instead show an error but the user won't be able to fix it since the field will be hidden.

Reply:
If that's the case @esirK , then are they really **required**? Seems to me like they should be _optional_.

Reply:
I agree. Makeing the fields optional (`blank=True, null=True`) results in this
![Screenshot 2020-09-17 at 09 07 47](https://user-images.githubusercontent.com/13068580/93426862-7dbcea80-f8c5-11ea-8178-00783150286d.png)
I'm sure we don't want this. I'm looking into why this is happening but in case you know how to remove the `none` I would really appreciate it.

Reply:
:thinking: ... The first 2 answers [here](https://stackoverflow.com/questions/7420241/how-to-remove-none-option-in-django-modelform) didn't work @esirK ?

Reply:
I did try the `blank=False` but the shortcoming was the form will now treat it as a required field and if we don't supply a value then it won't validate. I think I'll have to remove this requirement using javascript when the monitor mode is short.

Reply:
Cool if we can't find a Django-based solution quickly then I think why don't we make it optional in the backend and just hide the `None` option using JS @esirK? That way the backend will work with the `None` value and on the front-end, the user will only see `Yes` and `No` options...